tests/api/circulation/test_temporary_item_type.py

```
@@ -99,6 +103,331 @@ def test_checkout_temporary_item_type(
+    res, data = postdata(client, "api_item.checkin", checkin_params)
+    assert res.status_code == 200
+
     # reset the item to original value
     del item["temporary_item_type"]
     item.update(data=item, dbcommit=True, reindex=True)
+    ItemsSearch.flush_and_refresh()
+
+
+def test_checkout_remove_temporary_item_type_on_scan(
+    client,
+    document,
```

Ruff (0.14.13)

Unused function argument: `document`

(ARG001)